deps/wazuh_testing/wazuh_testing/tools/file.py

```diff
@@ -26,8 +26,11 @@
 from wazuh_testing import logger, REGULAR, SYMLINK, HARDLINK
 
 if sys.platform == 'win32':
+    import win32con
+    import win32api
     import win32security as win32sec
     import ntsecuritycon as ntc
+    import pywintypes
```

Comment:
This is necessary?

Reply:
Yes, the imports are necesary, pywintipes was missing when last refactored, since it is used on exceptions, if no exception is raised it is not noticed it's missing